### PR TITLE
Update website

### PR DESCRIPTION
**Pull request by fquinto**
Hello from "Barcelona Kernel Peer Lab" I like to introduce the group in your website.
Thanks!!

### CODE REVIEW

File: kernel-peer-lab.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+* Web page: https://bcnfs.org/kernel-peer-lab
+
+* Mailing List: https://lists.bcnfs.org/mailman/listinfo/kernel-peer-lab
+
+* IRC: libera.chat room [#bcn-kernel-peer-lab](https://web.libera.chat/#bcn-kernel-peer-lab)
+
+* Telegram group: https://t.me/+r-wnpzp59uQ5ODZk (missing: bridge to IRC/Matrix)
+
+* Matrix: The IRC room is available through matrix bridge
```
Comment (aleixpol) [inline kernel-peer-lab.md:25]: https://matrix.to/#/#bcn-kernel-peer-lab:libera.chat

File: index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
             <p class="introText">
                 <span class="introText--title">Barcelona Free Software</span> is a creators community of Free Software, also known as Open Source, in Barcelona. Our main activity is a <a class="u-fontMono" href="https://gettogether.community/barcelona-free-software/">Get Together</a> / <a class="u-fontMono" href="http://www.meetup.com/{{ site.meetup_username }}/">Meetup group</a>, an open gathering for developers and users who share a love for software freedom, but sometimes we organize bigger events too (like the <a href="https://2019.linuxappsummit.org/">Linux App Summit 2019</a>).
             </p>
-            <p class="introText">Come to our next meeting to discover new projects and have a drink.</p>
+            <p class="introText">Come to our next meeting to discover new projects or <a href="/groups">groups</a> and have a drink.</p>
```
Comment (aleixpol) [inline index.html:13]: How about a new line with:
`<p class="introText">Are you interested in Linux kernel programming? Learn more about our <a href="/kernel-peer-lab.md">Kernel peer lab</a>.</p>`

And then we don't need groups.md.

Comment (fquinto) [reply index.html:13]: It's ok. Groups was created in order to easily add more teams here.
Can you finish this changes? thanks!

Comment (aleixpol) [reply index.html:13]: TBH it's a bit of a shitty way to respond to a comment. But sure, I'll do it.

Comment (aleixpol) [reply index.html:13]: https://github.com/bcnfs/bcnfs.github.io/pull/50

**Pull request closed without merging**